proto/frequenz/api/microgrid/microgrid.proto

```diff
@@ -1,4 +1,69 @@
-// Contains the definitions for Microgrid gRPC API.
+// Frequenz Microgrid API
+//
+// Overview:
```

Comment:
This is what we removed from the proto definition and moved it just into the readme file. Of course we can replicate here too but in that case we should do that in all API definitions.

Reply:
In my tests, adding it here did not get it rendered in the web docs. I think it would be nice if the text appears when users open up the page for the `microgrid` proto file. That's why I added the whole text as the service documentation (so that it appears in the service docs).

Reply:
As discussed IRL, I'll remove it from here for now, and just keep it in README.md.